Formula/libmarpa.rb

```diff
@@ -26,6 +26,7 @@ class Libmarpa < Formula
 
   def install
     ENV.deparallelize
+    inreplace "work/etc/libmarpa.pc.in", "prefix=\".\"", "prefix=\"@CMAKE_INSTALL_PREFIX@\"" if build.head?
```

Comment:
```suggestion
    inreplace "work/etc/libmarpa.pc.in", "prefix=\".\"", "prefix=\"#{prefix}\"" if build.head?
```
Any reason not to just do this?

Reply:
I don't know how to decide which is better, is all.  If the patch was made upstream, I'd be making the replacement in my original PR.